flask_rebar/rebar.py

```diff
@@ -617,7 +617,7 @@ def _register_swagger_ui(self, app):
         blueprint_name = "swagger_ui"
 
         if self.prefix:
-            blueprint_name = self.prefix + blueprint_name
+            blueprint_name = self.prefix.replace(".", "") + blueprint_name
```

Comment:
can we replace the '.' with '_' so there is still at least some delimiter to show we removed the period?

Other than that, this seems fine.

Reply:
makes sense to me. Will do.